laravel/Caddyfile

```diff
@@ -0,0 +1,17 @@
+example.com {
+
+    root ./public
+    log ./storage/logs/caddy-access.log
+    errors ./storage/logs/caddy-error.log
```

Comment:
This example can probably be simplified without `log` and `errors`. Also, is `root` necessary (i.e. is it common for laravel apps to be in a different folder called "public")?